[2025-08-16T19:16:22.114+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-16T19:16:22.156+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: orders_pipeline.create_external_table_if_not_exists manual__2025-08-16T19:16:20+00:00 [queued]>
[2025-08-16T19:16:22.168+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: orders_pipeline.create_external_table_if_not_exists manual__2025-08-16T19:16:20+00:00 [queued]>
[2025-08-16T19:16:22.168+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-16T19:16:22.187+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): create_external_table_if_not_exists> on 2025-08-16 19:16:20+00:00
[2025-08-16T19:16:22.195+0000] {standard_task_runner.py:64} INFO - Started process 161 to run task
[2025-08-16T19:16:22.200+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'orders_pipeline', 'create_external_table_if_not_exists', 'manual__2025-08-16T19:16:20+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/orders_pipeline.py', '--cfg-path', '/tmp/tmp127jt_7b']
[2025-08-16T19:16:22.202+0000] {standard_task_runner.py:91} INFO - Job 45: Subtask create_external_table_if_not_exists
[2025-08-16T19:16:22.279+0000] {task_command.py:426} INFO - Running <TaskInstance: orders_pipeline.create_external_table_if_not_exists manual__2025-08-16T19:16:20+00:00 [running]> on host 392a94efd1c1
[2025-08-16T19:16:22.426+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='orders_pipeline' AIRFLOW_CTX_TASK_ID='create_external_table_if_not_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-08-16T19:16:20+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-16T19:16:20+00:00'
[2025-08-16T19:16:22.427+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-16T19:16:22.456+0000] {credentials.py:1217} INFO - Found credentials in environment variables.
[2025-08-16T19:16:24.888+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-16T19:16:24.889+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-16T19:16:24.912+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=orders_pipeline, task_id=create_external_table_if_not_exists, run_id=manual__2025-08-16T19:16:20+00:00, execution_date=20250816T191620, start_date=20250816T191622, end_date=20250816T191624
[2025-08-16T19:16:24.952+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-08-16T19:16:24.983+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-16T19:16:24.986+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
